catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -182,18 +182,23 @@ public Brick clone() {
 
 	@Override
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
+		if (this instanceof SetBackgroundBrick) {
+			Sprite backgroundSprite = getSpriteToChange();
+			sequence.addAction(sprite.getActionFactory().createSetLookAction(backgroundSprite, look, wait));
+		} else {
+			sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
+		}
```

Comment:
Is this change really necessary?
The sprite should always be the correct sprite, either a normal sprite or a background.